pkg/common/config/config.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"io/ioutil"
+	log "github.com/sirupsen/logrus"
+)
+
+var initContainerTemplate string
+
+func init() {
+	bytes, err := ioutil.ReadFile("/etc/config/initContainer.yaml")
```

Comment:
Do we need to expose InitContainerTemplate to user? Any particular use case?

Reply:
At least, the image used in init container will be configured in the private cloud. And to avoid the other use case not in mind,  expose InitContainerTemplate.